Sunrise/ViewModels/ScannerViewModel.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import Commercetools
+import ReactiveCocoa
+import ObjectMapper
+
+class ScannerViewModel: BaseViewModel {
+
+    // Inputs
+    let scannedCode = MutableProperty("")
+
+    // Outputs
+    let errorTitle = NSLocalizedString("Could not scan", comment: "Scanning error title")
+    let capabilitiesError = NSLocalizedString("Your device is not capable of performing barcode scanning.", comment: "Capabilities scanning error")
+    let permissionError = NSLocalizedString("In order to scan products, you need to allow camera access for Sunrise app. Please go to settings and grant camera permission.", comment: "Camera permissions error")
```

Comment:
In addition to this descriptive message, we provide them a direct option to go to settings from the alert shown in the app.